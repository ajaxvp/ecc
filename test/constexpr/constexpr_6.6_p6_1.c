/* ISO: 6.6 (6); additional checks in integer constant expressions */

enum e1
{
    // invalid cast (to ptr type) for integer constant expression
    E1_V1 = (unsigned long long) (int*) 5,
    // valid cast (to ptr type) for integer constant expression
    E1_V2 = sizeof((unsigned long long) (int*) 5)
};
