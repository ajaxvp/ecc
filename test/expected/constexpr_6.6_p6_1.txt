ecc: error: [6:5] enumeration constant value must be specified by an integer constant expression
